### ISSUE TITLE
`Extra` field is single-line only.

### ISSUE THREAD

**Issue by dragonmaus** [zeapo/Android-Password-Store#447]
I was adding a new password entry in the mobile app today when I was blocked by the fact that I couldn't add a second line to the Extra field. This is frustrating because my personal passfile format requires multiple lines, but I will have to wait until I can get to my desktop to fix this entry.

I don't often add entries from the mobile side of things, so I do not know how long it's been this way. I do know that the last time I did, the Extra field was a proper multi-line field, so something has changed at some point.

**Comment by zidhuss** [comment]
Thanks for bringing this to my attention.

This is a regression caused by https://github.com/zeapo/Android-Password-Store/commit/0b9eeb27e764aab8dacf11d53bc22b2be3cf4614. Fixed in https://github.com/zeapo/Android-Password-Store/commit/e72038f6f167bf5efc3cf292267ae4c2b6313672.

**Comment by dragonmaus** [comment]
Thank *you* for the fast turnaround. I look forward to this percolating down to the APK repositories.

**Comment by reroman** [comment]
It looks like the issue came back again in version 1.3.1. Should I open a new issue?

**Comment by zidhuss** [comment]
@reroman yeah go ahead.
